sync-core/src/main/java/com/cloudant/sync/query/IndexCreator.java

```diff
@@ -71,6 +72,14 @@ private String ensureIndexed(final Index index) {
             return null;
         }
 
+        if (index.indexType.equalsIgnoreCase("text")) {
+            if (!IndexManager.ftsCompileOptionsAvailable(queue, database)) {
```

Comment:
Just call it `ftsAvailable` rather than exposing the implementation in the method name.